### DISCUSSION TITLE
Anyone have ScannerCapabilities from a full duplex scanner?

### DISCUSSION THREAD

**Discussion by markosjal** [alexpevzner/sane-airscan#227] [General]
Anyone have ScannerCapabilities from a full duplex scanner?

If so can you please post?

wondering how the simplex/duplex options come out in ScannerCapabilities

Is there a "scan:AdfSimplexInputCaps" and "scan:AdfDuplexInputCaps"

or is there only "scan:AdfDuplexInputCaps"

**Comment by alexpevzner** [comment]
Yes. Please, have a look here:

https://github.com/alexpevzner/eSCL-protocol-traces/blob/master/Kyocera-ECOSYS-M2040dn.log

**Reply by markosjal** [reply]
Thanks, great repo I am gleaning a lot of info from it.
